### ISSUE TITLE
ValueError in StringConverter

### ISSUE THREAD

**Issue by AbdealiJK** [coala-analyzer/coala#800]
If I do `coala --files=src/*.c --bears=SpaceConsistencyBear --save` and simply press `Enter` when it asks for an option for `use_spaces` it gives this warning: 
```
[WARNING][19:56:54] Unable to convert use_spaces to the desired data type.
```

along with an ugly looking traceback saying ValueError

```
Process Process-3:
Traceback (most recent call last):
  File "/home/ajk/.pyenv/versions/3.4.2/lib/python3.4/multiprocessing/process.py", line 254, in _bootstrap
    self.run()
  File "/home/ajk/.pyenv/versions/3.4.2/lib/python3.4/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/home/ajk/.pyenv/versions/3.4.2/lib/python3.4/site-packages/coalib/processes/BearRunning.py", line 555, in run
    control_queue)
  File "/home/ajk/.pyenv/versions/3.4.2/lib/python3.4/site-packages/coalib/processes/BearRunning.py", line 439, in run_local_bears
    filename)
  File "/home/ajk/.pyenv/versions/3.4.2/lib/python3.4/site-packages/coalib/processes/BearRunning.py", line 318, in run_local_bears_on_file
    filename)
  File "/home/ajk/.pyenv/versions/3.4.2/lib/python3.4/site-packages/coalib/processes/BearRunning.py", line 225, in run_local_bear
    **kwargs)
  File "/home/ajk/.pyenv/versions/3.4.2/lib/python3.4/site-packages/coalib/processes/BearRunning.py", line 148, in run_bear
    kwargs)
  File "/home/ajk/.pyenv/versions/3.4.2/lib/python3.4/site-packages/coalib/processes/BearRunning.py", line 76, in validate_results
    result_list = list(result_list)
  File "/home/ajk/.pyenv/versions/3.4.2/lib/python3.4/site-packages/coalib/misc/../../bears/spacing/SpaceConsistencyBear.py", line 33, in run
    if use_spaces:
  File "/home/ajk/.pyenv/versions/3.4.2/lib/python3.4/site-packages/coalib/misc/StringConverter.py", line 44, in __bool__
    raise ValueError
ValueError
```

**Comment by sils1297** [comment]
was about to file the same issue, we should probably rather not execute the
bear in those cases and make the warning a failure. This way bears have a
guarantee that they'll get the exact type.

2015-07-16 16:29 GMT+02:00 AbdealiJK <notifications@github.com>:
